mail_factory/urls.py

```diff
@@ -24,4 +23,4 @@
     url(r'^html_not_found/(?P<mail_name>.*)/$',
         html_not_found,
         name='mail_factory_html_not_found'),
-)
+]
```

Comment:
That doesn't work with django < 1.8 does it ? I have nothing against it, but in that case shouldn't you bump the requirements in the README ?